src/components/todoItem.js

```diff
@@ -16,7 +17,7 @@ export default class TodoItem extends React.Component {
 		return (
 			<li className={[
 				todo.completed ? "completed": "",
-				todo === viewStore.todoBeingEdited ? "editing" : ""
+				computed(() => todo === viewStore.todoBeingEdited ? "editing" : "").get()
```

Comment:
I would better use (expr)[http://mobxjs.github.io/mobx/refguide/expr.html] for that